packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -100,7 +100,8 @@ export class Span implements APISpan, ReadableSpan {
     parentSpanId?: string,
     links: Link[] = [],
     startTime?: TimeInput,
-    _deprecatedClock?: unknown // keeping this argument even though it is unused to ensure backwards compatibility
+    _deprecatedClock?: unknown, // keeping this argument even though it is unused to ensure backwards compatibility
+    initAttributes?: SpanAttributes
```

Comment:
Nit: Maybe we can call it just `attributes`, I guess it'll be clear from them being passed to the constructor that tese are the initial ones. The span constructor is not supposed to be used directly by users anyway, as it'll become a private API in #3597 (scheduled for 2.0)
```suggestion
    attributes?: SpanAttributes
```

Reply:
@pichlermarc agree. Renamed.